scripts/accumulator-server.py

```diff
@@ -115,6 +114,18 @@ def all_done():
 
 app = Flask(__name__)
 
+@app.route("/v1/updateContext", methods=['POST'])
+def record():
+    global ac
+    ac += "Got a /v1/updateContext";
+    return Response(status=200)
+
+@app.route("/v1/queryContext", methods=['POST'])
+def record():
+    global ac
+    ac += "Got a /v1/queryContext";
+    return Response(status=200)
+
```

Comment:
Now tell me why this 'stupid' modification makes the accumulator behave the way I want ...
I expected to see 'Got a /v1/updateContext' and 'Got a /v1/queryContext' in the dump, and after that keep working to get the message in 'ac' instead.
However, in the first test I got already the exact output I wanted from /dump. Magic ... ?

Also, I've seen that in some functions use declare 'ac' as global and in others you don't - confusing.